crates/storage/src/collections/stash/tests.rs

```diff
@@ -445,10 +445,11 @@ fn simple_defrag_works() {
 
 /// Returns a storage stash that looks internally like this:
 ///
-///        i | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 |
-///     next |   |   |   |   |   |   |   |   |
-/// previous |   |   |   |   |   |   |   |   |
-///      val |   |   |   |   | E |   |   | H |
+///   i        | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7
+///  ----------|---|---|---|---|---|---|---|---
+///      next  |   |   |   |   |   |   |   |
+///   previous |   |   |   |   |   |   |   |
+///        val |   |   |   |   | E |   |   | H
```

Comment:
What is the improvement here? :thinking:

Reply:
hunspell recognizes it as a table now…